organizations/tests.py

```diff
@@ -39,3 +48,57 @@ def test_superuser_can_list_organizations(self):
         self.user.save()
         response = self.client.get(reverse("organizations-list"))
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_updating_organization_recreates_user_instance_cache(self):
+        from users.models import User
+        from rest_framework.test import APIClient
+        from plio.tests import get_new_access_token
+
+        superadmin_client = APIClient()
+        superadmin_user = User.objects.create(mobile="+919988776655", is_superuser=True)
+        superadmin_access_token = get_new_access_token(
+            superadmin_user, self.application
+        )
+        superadmin_client.credentials(
+            HTTP_AUTHORIZATION="Bearer " + superadmin_access_token.token
+        )
+
+        # verify cache data doesn't exist by default
+        cache_key_name = get_cache_key(self.user)
```

Comment:
I think I didn't understand how does this line is verifying that cache data doesn't exist

Reply:
There should be this line here as well, right?
`self.assertEqual(len(cache.keys(cache_key_name)), 0)`

Reply:
Yes. Added. Thanks for catching.